library/std/math.qs

```diff
@@ -442,6 +441,7 @@ namespace Microsoft.Quantum.Math {
     /// Returns a tuple (u,v) such that u*a+v*b=GCD(a,b)
     /// Note: GCD is always positive except that GCD(0,0)=0.
     function ExtendedGreatestCommonDivisorL(a : BigInt, b : BigInt) : (BigInt, BigInt) {
+        open Microsoft.Quantum.Convert;
```

Comment:
Interesting that the open directive was moved here, I didn't know it could be done. Was there any particular reason to move it here?

Reply:
Conversions are only used in three functions, so I decided to move it to those functions.

Reply:
(I believe the ability to put open directive in any block is the feature of the new compiler)